### PR TITLE
Update string assert function

### LINKED ISSUE #11: Exception with address line
Hi, 

I noticed a problem creating payment form. If I put this address `2 avenue de la botte aux bailles, Bâtiment Pasaïa` I get an exception for invalid address.

| String  | Status | Length(mb_strlen) |
| ------------- | ------------- | ------------- |
| `2 avenue de la botte aux bailles, Bâtiment Pasaïa`  | KO  ❌ | 49 |
| `2 avenue de la botte aux bailles, Bâtiment Pasaï`  | OK  ✅ | 48 |
| `2 avenue de la botte aux bailles, Bâtiment Pasaia`  | OK ✅ | 49 |
| `Bâtiment Pasaïa`  | OK ✅ | 15 |

The problem is string assert function that use `strlen` instead of `mb_strlen`
I made a PR with test & update.

### PR DESCRIPTION
**Pull request by ehibes**
Resolve #11

### CODE REVIEW

File: tests/Api/ApiTest.php [hunk 1]
```diff
@@ -173,4 +173,32 @@ private function createApi($mode = Api::MODE_PRODUCTION)
...
+            'currency'    => 'EUR',
+            'reference'   => '100008783',
+            'comment'     => 'Commande 100008783',
+            'locale'      => 'FR',
+            'email'       => 'test@example.org',
+            'success_url' => 'http://example.org',
+            'failure_url' => 'http://example.org',
+            'context'     => [
+                'billing' => [
+                    'addressLine1'    => '2 avenue de la botte aux bailles, Bâtiment Pasaïa',
```
Comment (Prometee) [inline tests/Api/ApiTest.php:193]: @ehibes maybe adding a full non iso string here can help testing the 50 char limit :
```suggestion
                    'addressLine1'    => 'ïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïïï',
```

File: composer.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "php-http/guzzle7-adapter": "^1.0",
-        "phpunit/phpunit": "^8.0|^9.0"
+        "phpunit/phpunit": "^8.0|^9.0",
+        "php-http/message-factory": "^1.1"
```
Comment (Prometee) [inline composer.json:30]: @ekyna this requirement has been added to wait for a proper fix from Payum since this packages is deprecated

**Review by Prometee — APPROVED** [review]

**Comment by Prometee** [comment]
@ekyna ready to merge 😉

**Comment by ekyna** [comment]
Released 1.7.2
Thx @ehibes and @Prometee.

**Pull request merged**